src/main/java/RandomNumberGenerator.java

```diff
@@ -0,0 +1,10 @@
+import java.util.Random;
+
+public class RandomNumberGenerator implements NumberGenerator{
+    private static final int RANDOM_NUMBER_BOUND = 10;
+    private static final Random random = new Random();
+    @Override
+    public int generate(){
```

Comment:
```suggestion
    private static final Random random = new Random();

    @Override
    public int generate(){
```

개행으로 더 보기 좋은 코드로 만들수 있어요!